src/connections/destinations/catalog/actions-stackadapt/index.md

```diff
@@ -0,0 +1,116 @@
+
+![Image showing event rule in StackAdapt the matches a User Registered event](images/user-registered-event-rule.png)
+
+#### Ecommerce Events
+
+The StackAdapt destination also supports forwarding ecommerce fields for the purpose of creating event rules that match ecommerce events, with default mappings to properties specified in the [Segment V2 Ecommerce Event Spec](/docs/connections/spec/ecommerce/v2/) as described in the below table:
+
+| Segment Ecommerce Event Property | StackAdapt Event Key |
+|----------------------------------|----------------------|
+| `order_id`                       | `order_id`           |
+| `revenue`                        | `revenue`            |
+| `product_id`                     | `product_id`         |
+| `category`                       | `product_category`   |
+| `name`                           | `product_name`       |
+| `price`                          | `product_price`      |
+| `quantity`                       | `product_quantity`   |
+
+For events that can involve multiple products, such as checkout events, we forward a JSON array of product objects with a `products` key and fields that map by default to following Segment product array fields:
+
+| Segment Ecommerce Event Property | StackAdapt Product Object Key |
+|----------------------------------|-------------------------------|
+| `products.$.product_id`          | `product_id`                  |
+| `products.$.category`            | `product_category`            |
+| `products.$.name`                | `product_name`                |
+| `products.$.price`               | `product_price`               |
+| `products.$.quantity`            | `product_quantity`            |
+
+For example, to create a conversion event when an order is completed with a revenue value greater than 10, you could set up an event rule matching an `action` value of `Order Completed` and a `revenue` value greater than 10 as shown below:
+
+![Image showing event rule in StackAdapt the matches an Order Completed event with a revenue greater than 10](images/order-completed-event-rule.png)
+
+This rule would match a Segment event fired with code such as:
+
+```javascript
+analytics.track("Order Completed", {
+  order_id: '50314b8e9bcf000000000000',
+  revenue: 11.5
+  products: [
+    {
+      product_id: '507f1f77bcf86cd799439011',
+      name: 'Monopoly: 3rd Edition',
+      price: 11.5,
+      quantity: 1,
+      category: 'Games'
+    }
+  ]
+});
+```
+
+### URL Rules
+
+If you are using URL rules, these will be matched whenever Segment sends an event to StackAdapt with a `url` matching the URL rule. This should be accomplished by the page event Segment automatically fires when a page is viewed, so setup of URL rules should be identical to setting up URL rules with the StackAdapt pixel.
+
+{% include components/actions-fields.html %}
```

Comment:
Segment requests we include this component in their [documentation template](https://raw.githubusercontent.com/segmentio/action-destinations/main/docs/doc-template-new.md). It doesn't render properly currently but as I understand it, it should once our destination is public.